packages/salesforcedx-apex-replay-debugger/src/core/logContext.ts

```diff
@@ -99,6 +113,88 @@ export class LogContext {
     );
   }
 
+  public getHeapDumps(): ApexHeapDump[] {
+    return this.apexHeapDumps;
+  }
+
+  public scanLogForHeapDumpLines(): boolean {
```

Comment:
This matches the "|HEAP_DUMP|" log tags. If it finds one it splits the line and sets all of the information. Since logging changes more or less have to be additive the ordering here isn't going to change.